go.mod

```diff
@@ -1,5 +1,10 @@
 module github.com/josharian/impl
 
-go 1.14
+go 1.19
 
 require golang.org/x/tools v0.0.0-20200522201501-cb1345f3a375
+
+require (
```

Comment:
May as well run `go get -u ./...` and upgrade all the deps while we're at it?

Reply:
I generally like the dependency updates introduced in a PR to only be the updates required to support the functionality being used. I think Dependabot is a better answer for how to keep the module dependencies up to date with the latest versions, if that's a desirable characteristic for a codebase. Or doing updates not required by new functionality in their own, dedicated PRs. Which is essentially what Dependabot is automating.

That said, that's more a maintenance philosophy than anything else, so if how you want to maintain your repo is to just bump the dependencies as part of this PR, I can do that and won't backseat maintain. Just wanted to give you the info for why it was done this way. If you'd like me to go ahead and run that, just comment on this chain letting me know that you still want that course of action, and I'll do it. 👍

Reply:
Upgraded deps and pushed.